torch/distributed/fsdp/_runtime_utils.py

```diff
@@ -53,6 +54,37 @@ class _PrefetchMode(Enum):
     FORWARD = auto()
 
 
+def fsdp_state_has_default_pg(state: _FSDPState) -> bool:
+    """Indicates whether FlatParamHandle has the default process group.
+
+    Args:
+        handle (_FSDPState): FSDP State object
```

Comment:
nit: This looks like it has a typo. I would be good with not including this comment at all since the code seems self-explanatory.

Reply:
done!